guides/v2.1/config-guide/config/config-php.md

```diff
@@ -28,6 +28,18 @@ Magento's deployment configuration consists of the shared and system-specific co
 	`config.php` contains the list of installed modules, themes, and language packages; and shared configuration settings
 *	`<Magento base dir>/app/etc/env.php`, which contains system-specific settings, such as:
 
+	*	Database credentials and connection settings	
+	*	i18n TBD	
+	*	Cache storage settings	
+	*	Session storage settings	
+	*	[x-frame-options]({{ page.baseurl }}config-guide/secy/secy-xframe.html) setting	
```

Comment:
One minor change that I will go ahead and make... for `page.baseurl` URLs, you have to add a `/` after `}}` to make it work.